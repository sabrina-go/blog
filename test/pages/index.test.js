import { mount } from '@vue/test-utils';
import Index from '@/pages/index';

describe('page | index', () => {
  let page;
  beforeEach(() => {
    page = mount(Index);
  });

  test('is a Vue instance', () => {
    expect(page.vm).toBeTruthy();
  });

  test('render properly', () => {
    expect(page.html()).toMatchSnapshot();
  });
});
