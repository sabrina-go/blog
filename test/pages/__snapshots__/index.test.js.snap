// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page | index render properly 1`] = `
"<div class=\\"index\\">
  <div class=\\"index__container\\">
    <h1 class=\\"index__title\\">blog-nuxt</h1>
    <div class=\\"index__articles\\"></div>
    <div v-if=\\"[object Object]\\" class=\\"index__articles-remaining\\">
      <articlecard-stub article=\\"[object Object]\\" class=\\"index__article-remaining\\"></articlecard-stub>
    </div>
  </div>
</div>"
`;
